Python/structmember.c

```diff
@@ -69,8 +69,13 @@ PyMember_GetOne(const char *addr, PyMemberDef *l)
         break;
     case T_OBJECT_EX:
         v = *(PyObject **)addr;
-        if (v == NULL)
-            PyErr_SetString(PyExc_AttributeError, l->name);
+        if (v == NULL) {
+            PyObject *obj = (PyObject *)obj_addr;
+            PyTypeObject *tp = Py_TYPE(obj);
+            PyErr_Format(PyExc_AttributeError,
+                         "'%.50s' object has no attribute '%s'",
```

Comment:
I thought .200s was used everywhere. I get 349 hits on ".200s", and 25 hits on ".50s". Not that it's very important.

Reply:
Fixed in the last commit